.jenkins/pipeline.jenkins

```diff
@@ -0,0 +1,686 @@
+      //    steps {
+      //      testJob(".build/docker/run-tests.sh", "test", 8)
+      //    }
+      //  }
+      //  stage('long units') {
+      //    steps {
+      //      testJob(".build/docker/run-tests.sh", "long-test", 8)
+      //    }
+      //  }
+      //  stage('burn') {
+      //    steps {
+      //      testJob(".build/docker/run-tests.sh", "test-burn", 8)
+      //    }
+      //  }
+      //  stage('cdc') {
+      //    steps {
+      //      testJob(".build/docker/run-tests.sh", "test-cdc", 8)
+      //    }
+      //  }
+      //  stage('compression') {
+      //    steps {
+      //      testJob(".build/docker/run-tests.sh", "test-compression", 8)
+      //    }
+      //  }
+      //  stage('cqlsh') {
+      //    steps {
+      //      testJob(".build/docker/run-tests.sh", "cqlsh-test", 1)
+      //    }
+      //  }
+      //  stage('jvm-dtest') {
+      //    steps {
+      //      testJob(".build/docker/run-tests.sh", "jvm-dtest", 8)
+      //    }
+      //  }
+      //  stage('jvm-dtest-upgrade') {
+      //    steps {
+      //      testJob(".build/docker/run-tests.sh", "jvm-dtest-upgrade", 8)
+      //    }
+      //  }
+      //  stage('dtest') {
+      //    steps {
+      //      testJob(".build/docker/run-tests.sh", "dtest", 64)
+      //    }
+      //  }
+      //  stage('dtest-large') {
+      //    steps {
+      //      testJob(".build/docker/run-tests.sh", "dtest-large", 8)
+      //    }
+      //  }
+      //  stage('dtest-novnode') {
+      //    steps {
+      //      testJob(".build/docker/run-tests.sh", "dtest-novnode", 64)
+      //    }
+      //  }
+      //  stage('dtest-offheap') {
+      //    steps {
+      //      testJob(".build/docker/run-tests.sh", "dtest-offheap", 64)
+      //    }
+      //  }
+      //  stage('dtest-large-novnode') {
+      //    steps {
+      //      testJob(".build/docker/run-tests.sh", "dtest-large-novnode", 8)
+      //    }
+      //  }
+      //  stage('dtest-upgrade') {
+      //    steps {
+      //      testJob(".build/docker/run-tests.sh", "dtest-upgrade", 64)
+      //    }
+      //  }
+     }
+    }
+    stage('Summary') {
+      steps {
+        sh 'echo "stage summary"'
+        dir("build") {
+          unstash name: "cassandra_11_stress-test"
+          //unstash name: "cassandra_11_fqltools"
+        }
+       //container('jnlp') {
+       // FIXME is this needed if each job is submitting its own results to the overall pipeline ??
+       //junit testResults: '**/build/test/**/TEST*.xml,**/cqlshlib.xml,**/nosetests.xml', testDataPublishers: [[$class: 'StabilityTestDataPublisher']]
+       
+       //sendNotifications()
+       
+       // FIXME how to collect and build one test report file ?? 
+       //generateUnifiedTestReport()
+       
+        //collectConsoleLog() // FIXME should this be in a final instead ? (we always want the console log archived)
+        //copyToNightlies('console.log.xz,TESTS-TestSuites.xml.xz')
```

Comment:
i think only things that remains here is
```
sendNotifications()
copyToNightlies('console.log.xz,TESTS-TestSuites.xml.xz')
```
(both of which won't do anything in a k8s env, they are only intended to work when `isCanonical()`)